client/src/components/Workflow/WorkflowActionsExtend.vue

```diff
@@ -72,11 +65,41 @@ async function onRestore() {
         Toast.info("Workflow restored");
     }
 }
+
+const root = computed(() => {
+    const port = window.location.port ? `:${window.location.port}` : "";
+    return `${window.location.protocol}//${window.location.hostname}${port}${getAppRoot()}`;
+});
+
+const relativeLink = computed(() => {
+    return `/published/workflow?id=${props.workflow.id}`;
+});
+
+const fullLink = computed(() => {
+    return `${root.value}${relativeLink.value.substring(1)}`;
+});
+
+function onCopyPublicLink() {
+    copy(fullLink.value);
+    Toast.success("Link to workflow copied");
+}
 </script>
 
 <template>
     <div class="workflow-actions-extend flex-gapx-1">
         <BButtonGroup>
+            <BButton
+                v-if="!isAnonymous && workflow.published && !workflow.deleted"
```

Comment:
Does this mean anonymous users can't copy the link ? Why is that ?

Reply:
Good catch! The condition was copied from another button and wasn’t necessary here, so I’ve removed it. Thanks for pointing it out.